tools/arm_pack_manager/index.json

```diff
@@ -302994,12 +302994,7 @@
                 "units": 1
             }
         }, 
-        "sectors": [
-            [
-                0, 
-                4096
-            ]
-        ], 
+        "sectors": [], 
```

Comment:
You need to have some information here, it should be the sector start address and size. Even one entry like previously would do, but in reality the start should be 536870912 and size should be the erase sector size (HW reference should tell that).

Travis CI fails otherwise.

Same for the other board, too.

Reply:
Is it necessary even if only RAM is defined? It seems that other devices do not mention the RAM sector information.
ROM start address, ROM size and ``sectors`` have been added to ``targets.json``.
If ``sectors`` is described in both ``index.json`` and ``targets.json``, does ``targets.json`` take precedence?

Reply:
In an idea world, we .json file would be enough, but unfortunately we're not there. RAM does not have to be defined for the sectors, only the flash.